lib/axon/activations.ex

```diff
@@ -410,7 +410,7 @@ defmodule Axon.Activations do
   @doc ~S"""
   Log-sigmoid activation.
 
-  $$f(x_i) = \log(\sigmoid(x))$$
+  $$f(x_i) = \log(\sigma(x))$$
```

Comment:
I think the correct fix would be:
```suggestion
  $$f(x_i) = \log(sigmoid(x))$$
```

Reply:
I can change it if you think that's clearer. The sigmoid function can be denoted as σ so I assumed that was the intent here.

Reply:
This module exposes a `sigmoid` function, so I think `sigmoid` is the clearer alternative with that in mind

Reply:
There's also `Nx.sigmoid` that uses the full name as well

Reply:
Good point, I amended it.